wagtail/project_template/project_name/templates/base.html

```diff
@@ -8,7 +8,7 @@
     <head>
         <meta charset="utf-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <title>{% templatetag openblock %} block title %}{% templatetag openblock %} if self.seo_title %}{% templatetag openvariable %} self.seo_title {% templatetag closevariable %}{% templatetag openblock %} else %}{% templatetag openvariable %} self.title {% templatetag closevariable %}{% templatetag openblock %} endif {% templatetag closeblock %}{% templatetag openblock %} endblock {% templatetag closeblock %}{% templatetag openblock %} block title_suffix {% templatetag closeblock %}{% templatetag openvariable %} self.get_site.site_name {% templatetag closevariable %}{% templatetag openblock %} endblock {% templatetag closeblock %}</title>        
+        <title>{% templatetag openblock %} block title %}{% templatetag openblock %} if self.seo_title %}{% templatetag openvariable %} self.seo_title {% templatetag closevariable %}{% templatetag openblock %} else %}{% templatetag openvariable %} self.title {% templatetag closevariable %}{% templatetag openblock %} endif {% templatetag closeblock %}{% templatetag openblock %} endblock {% templatetag closeblock %} - {% templatetag openblock %} block title_suffix %}{% templatetag openblock %} if self.get_site.site_name %}{% templatetag openvariable %} self.get_site.site_name {% templatetag closevariable %}{% templatetag openblock %} else %}{% templatetag openvariable %} self.get_site.hostname {% templatetag closevariable %}{% templatetag openblock %} endif {% templatetag closeblock %}{% templatetag openblock %} endblock {% templatetag closeblock %}</title>
```

Comment:
the `%} - {% templatetag openblock %} block title_suffix %}{% templatetag openblock %}` bit has an extraneous `%}` after block_title_suffix